addons/project/tests/test_project_recurrence.py

```diff
@@ -102,23 +102,18 @@ def test_recurrence_until(self):
             form.name = 'test recurring task'
             form.project_id = self.project_recurring
             form.recurring_task = True
-            form.repeat_interval = 5
+            form.repeat_interval = 1
             form.repeat_unit = 'month'
             form.repeat_type = 'until'
-            form.repeat_until = self.date_01_01 + relativedelta(months=1)
+            form.repeat_until = self.date_01_01 + relativedelta(months=1, days=1)
+            form.date_deadline = self.date_01_01
             task = form.save()
 
-        with freeze_time(self.date_01_01 + relativedelta(days=30)):
-            task.state = '1_done'
```

Comment:
Why did you delete that?

Reply:
The current date doesn't  matter anymore since we're only comparing the task's date(s) to the recursion limit (Unless we go with nle's suggestion). Previously we used to compare date.today() to the limit instead.

Reply:
Euh you're right :ok_hand: